src/io_threads.c

```diff
@@ -501,7 +503,12 @@ int tryOffloadFreeObjToIOThreads(robj *obj) {
         return C_ERR;
     }
 
-    IOJobQueue_push(jq, decrRefCountVoid, obj);
+    /* We offload only the free of the ptr that may be allocated by the I/O thread.
+     * The object itself was allocated by the main thread and will be freed by the main thread. */
+    IOJobQueue_push(jq, zfree, obj->ptr);
```

Comment:
In OBJ_ENCODING_RAW, the pointer is an sds string, right?

```suggestion
    IOJobQueue_push(jq, sdsfree, obj->ptr);
```

Reply:
Yes this is an error, I fixed it but forgot to commit, fixed. Thanks!